misc/dockerfiles/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+echo_green() { echo -e "\e[42m$@\e[0m"; }
+echo_red()   { echo -e "\e[101m$@\e[0m"; }
```

Comment:
Shell escapes are typically... difficult to do in a standardized way - here, green is unreadable and red cuts of chars:

![image](https://user-images.githubusercontent.com/625793/37649670-c8d067ca-2c32-11e8-9a11-7cc130bf2845.png)

Not sure if it's really worth doing this - I'd rather have a simpler script without colors but readable output instead :wink:

Reply:
The problem ist, that you can either supress all build output, which can be confusing if you have changes that take a while (like `pipenv sync`). It's not clear why it takes that long.

If you print all the build output you need to group the output so that it's clear what is done where.

You're right, colours are not perfect, but it's a simple way to group them. As long as I'm the only one using it I don't think it's necessary to over-engineer this script